leash.go

```diff
@@ -113,9 +113,9 @@ func run(options GlobalOptions) {
 			close(realToBeSent)
 		}()
 
-		// start up the sender
-		go sendToLibhoney(realToBeSent, toBeResent, delaySending, doneSending,
-			options.TailSample)
+		// start up the sender. all sources are either sampled when tailing or in-
+		// parser, so always tell libhoney events are pre-sampled
+		go sendToLibhoney(realToBeSent, toBeResent, delaySending, doneSending, true)
```

Comment:
if this is the only invocation of `sendToLibHoney` and we always pass in `true`, is... it worth keeping that `presampled bool` param in place? are we likely to want to use it again in the future?